src/Interpreters/Cache/FileSegment.h

```diff
@@ -56,6 +56,13 @@ friend class FileCache; /// Because of reserved_size in tryReserve().
     using Info = FileSegmentInfo;
     using QueueEntryType = FileCacheQueueEntryType;
 
+    enum SizeAlignment
+    {
+        CACHE_ALIGNMENT,
+        ALIGNED,
+        NOT_ALIGNED
+    };
```

Comment:
`CACHE_ALIGNMENT` -> `DEFAULT` and comments please :)
```
   /// A hint for what size to use in methods like `getSize()`.
   /// By default, the DEFAULT kind should be used,
   /// but a specific kind (aligned/non-aligned) of size might be requested explicitly in case of assertions.
   enum SizeAlignment
    {
        /// Use the default size alignment,
        /// which is chosen according to `use_real_cache_size` setting.
        DEFAULT,
        /// Use aligned size.
        ALIGNED,
        /// Use non-aligned size.
        NOT_ALIGNED
    };
```